Header header

string robot_ns
geometry_msgs/Pose pose
float64 accum_distance
sensor_msgs/PointCloud2 cloud
float64[4] floor_coeffs
